src/flow-bin/README.md

```diff
@@ -68,25 +68,25 @@ cache:
 flow:
   # ...
   script:
-    - yarn run kiwicom-flow-bin
+    - yarn run adeira-flow-bin
 ```
 
 # Technical details
 
 Complete re-check (`restart --all`) will take a long time. We run many optimizations by default. Notably we are running Flow in Lazy Mode and we are trying to re-use Saved State when possible. Lazy Mode and Saved State are currently not configurable from outside since it should not be needed. Here are some approximate example timings (on 12356 relevant files):
 
 ```text
-yarn kiwicom-flow-bin (first run)      ~13s      (cold start, lazy mode, no saved state) ❌
-yarn kiwicom-flow-bin (second run)     ~3s       (hot start, lazy mode, saved state) ✅
-yarn kiwicom-flow-bin restart          ~8s       (cold start, lazy mode, saved state) ❌
-yarn kiwicom-flow-bin restart --all    ~22s      (cold start, no lazy mode, no saved state) ❌
+yarn adeira-flow-bin (first run)      ~13s      (cold start, lazy mode, no saved state) ❌
+yarn adeira-flow-bin (second run)     ~3s       (hot start, lazy mode, saved state) ✅
+yarn adeira-flow-bin restart          ~8s       (cold start, lazy mode, saved state) ❌
+yarn adeira-flow-bin restart --all    ~22s      (cold start, no lazy mode, no saved state) ❌
```

Comment:
The alignment is a bit broken here (1 missing space)

Reply:
I don't see it, where?

Reply:
Ahh, ok, found it in IDE.